src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -59,8 +59,8 @@ fi
 npm run build ${FRONTEND_BUILD_OPTS} &
 
 cd ${CEPH_DEV_FOLDER}
-: ${VM_IMAGE:='fedora36'}
-: ${VM_IMAGE_URL:='https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/36/Cloud/x86_64/images/Fedora-Cloud-Base-36-1.5.x86_64.qcow2'}
+: ${VM_IMAGE:='fedora40'}
+: ${VM_IMAGE_URL:='https://download.fedoraproject.org/pub/fedora/linux/releases/40/Cloud/x86_64/images/Fedora-Cloud-Base-Generic.x86_64-40-1.14.qcow2'}
```

Comment:
Is this required for the PR ?

Reply:
it'd improve the speed so yeah, it is. and since I am upgrading the version in `ceph_cluster.yml` file, this partivular image should exist in the network pool we create in jenkins.

Reply:
f36 is very old and we are pulling it from the archives which is even slower when downloading it. And refreshing the dependencies in f36 is significantly slower for me atleast and it was improved when I used f40. Which is why I went ahead with the change. If we want we can have it done separately but I didn't see the need for it as it is improving the speed for me locally. If its not the case in your testing, then I can remove this change and do it separately.